src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/FlowHelper.groovy

```diff
@@ -130,16 +130,7 @@ class FlowHelper {
     FlowPayload addFlow(FlowPayload flow) {
         log.debug("Adding flow '${flow.id}'")
         def response = northbound.addFlow(flow)
-
-        def flowEntry = null
-        Wrappers.wait(WAIT_OFFSET) {
-            assert northbound.getFlowStatus(flow.id).status == FlowState.UP
-
-            flowEntry = db.getFlow(flow.id)
-            assert flowEntry
-        }
-        checkRulesOnSwitches(flowEntry, RULES_INSTALLATION_TIME, true)
```

Comment:
previously in v1 system was not able to guarantee rules installation before setting the flow to UP, so we had to wait for them ourselves after creating each flow. In h&s now we don't have to do this for each flow, since it is checked by the system. Instead, we have a separate test that verifies that system checks this correctly for us